controller/schema.go

```diff
@@ -241,6 +241,45 @@ $$ LANGUAGE plpgsql`,
 	migrations.Add(10,
 		`ALTER TABLE formations ADD COLUMN tags jsonb`,
 	)
+	migrations.Add(11,
+		`ALTER TABLE artifacts ADD COLUMN attributes jsonb`,
+		`CREATE TABLE release_tar_artifacts (
```

Comment:
Ok I wasn't super happy with this either, I'll play around see what I come up with.